_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -16,9 +16,13 @@ Please note that the general procedure is the same if the Connector wants to rea
 
 The Sender wants to read an Attribute of the Recipient. To do this, the Sender must first create a suitable [Request]({% link _docs_integrate/data-model-overview.md %}#request), which it can then send to the Recipient. In the following subsections, we describe the general appearance of a Request for reading Attributes.
 
+### Role of ReadAttributeRequestItem
+
+For reading a single Attribute, the Sender needs to insert a single [RequestItem]({% link _docs_integrate/data-model-overview.md %}#requestitem) of type [ReadAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem) into the `items` property of the [Request]({% link _docs_integrate/data-model-overview.md %}#request). The input it has to provide for the `query` property of the [ReadAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem) depends on what type of Attribute it wants to get. If the Sender wants to read an IdentityAttribute, it must use an appropriate [IdentityAttributeQuery]({% link _docs_integrate/data-model-overview.md %}#identityattributequery). If it is interested in a RelationshipAttribute of the Recipient, it must insert a [RelationshipAttributeQuery]({% link _docs_integrate/data-model-overview.md %}#relationshipattributequery) or a [ThirdPartyRelationshipAttributeQuery]({% link _docs_integrate/data-model-overview.md %}#thirdpartyrelationshipattributequery) at this point. This depends on whether the Sender wants to read a RelationshipAttribute that the Recipient has in the context of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with the Sender or with a third party.
```

Comment:
`[RequestItem]({% link _docs_integrate/data-model-overview.md %}#requestitem)` will be removed soon.

Reply:
Did you mean you would change the link to `...#requestitem**s**`, remove the link completely or avoid using the term RequestItem at this point?

Reply:
I think removing would be the best option as you link to a specific item directly.